Deathmatch

TheRoboticon opened the pull request:
This command follows the format ?deathmatch [username] and challenges the specified user to a 'deathmatch' fight decided randomly. The 4 possibilities are typeracer, roulette, coin flip, and insult. As it might have repercussions for pinging people, I suggest the implementation of a role that restricts the user or receiver of the command. If so, I will edit the command personally to include this addition.

advaith1 commented:
rejecting as requested by redstonedesigner because of a bug un the update commandban commit

Commit:
Deathmatch

This command follows the format ?deathmatch [username] and challenges the specified user to a 'deathmatch' fight decided randomly. The 4 possibilities are typeracer, roulette, coin flip, and insult. As it might have repercussions for pinging people, I suggest the implementation of a role that restricts the user or receiver of the command. If so, I will edit the command personally to include this addition.
